taxi_service/settings.py

```diff
@@ -43,6 +43,10 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "debug_toolbar",
+
+    "crispy_forms",
+    "crispy_bootstrap4",
```

Comment:
The `crispy_forms` and `crispy_bootstrap4` apps are correctly added to `INSTALLED_APPS`, which is necessary for using crispy forms with Bootstrap 4.